build/griddle.js

```diff
@@ -79,6 +79,10 @@ return /******/ (function(modules) { // webpackBootstrap
 	var _ = __webpack_require__(3);
 
 	var Griddle = React.createClass({displayName: "Griddle",
+	    propTypes: {
```

Comment:
@ryanlanciaux note that I've added in `propTypes` but only for the new property. Would be nice to have these set for all properties in the future. I'm happy to add in if you'd like.